### ISSUE TITLE
Inconsistency in compression between Windows and MacOS 

### ISSUE THREAD

**Issue by stravik** [kornelski/pngquant#310]
Hello,

Running the binaries of `pngquant` on Windows produces a different result than on MacOS, resulting in unwanted changes in the git repository.

Is it possible to customize the settings to get the same compression result? I tried with and without `--strip`.

Downloaded the latest version for both from the website today. It says both are `2.12.0`, but they show different versions when running `pngquant --version`.

- Windows: `2.11.7`
- MacOS: `2.12.0`

Thanks!

**Comment by kornelski** [comment]
No, sorry. I can't guarantee cross-platform consistency. I don't even guarantee consistency on the same platform.

Some parts of the algorithm use floating-point math and allow compilers to optimize it imprecisely, so different versions of the code may give slightly different results.

Different platforms use different libraries for handling color profiles, and that's another source of small variations.
